Kros.ProjectTemplates/src/Kros.CqrsTemplate/content/Program.cs

```diff
@@ -24,10 +24,17 @@ public static void Main(string[] args)
         /// <param name="args">Arguments.</param>
         public static IHostBuilder CreateHostBuilder(string[] args) =>
             Host.CreateDefaultBuilder(args)
-                .AddLocalConfiguration()
-                .ConfigureWebHostDefaults(webBuilder => 
-                { 
-                    webBuilder.UseStartup<Startup>(); 
-                }); 
+                .ConfigureAppConfiguration((hostingContext, config) =>
```

Comment:
In other services we use: 

```CSharp
.ConfigureAppConfiguration((hostingContext, config) =>
                {
                    var settings = config.Build();
                    string appConfigEndpoint = settings["AppConfig:Endpoint"];
                    if (!string.IsNullOrWhiteSpace(appConfigEndpoint))
                    {
                        config.AddAzureAppConfiguration(hostingContext);
                    }
                    config.AddLocalConfiguration();
                })
```

without environment check.

Reply:
Done.